acstools/acsphotcte.py

```diff
@@ -0,0 +1,289 @@
+#!/usr/bin/env python
+"""
+The ACS Photometric CTE API is a programmatic interface for the
+`ACS Photometric CTE Webtool <https://acsphotometriccte.stsci.edu>`_.
+The API is a cloud-based service that employs a serverless approach on AWS
+with API Gateway and Lambda to compute the photometric CTE corrections
+using the model described in ACS ISR 2012-02.
+The model corrects ACS/WFC aperture photometry extracted from FLT images for
+CTE losses using the most recent formula from Chiaberge's ISR 2012-05. It is
+only calibrated for photometry obtained after SM4 in May 2009. For pre-SM4
+data, please see Chiaberge et al. 2009 (ISR 2009-01), or use pixel-based
+CTE-corrected files obtained from MAST. Currently,  only 3 and 5
+pixel aperture radii are supported. The model is designed to compute the
+CTE losses as a function of the number of Y transfers, sky background,
+source flux, and observation date.
+
+
+Usage Guidelines
+----------------
+As this service is hosted on AWS, there are some guidelines that should be
+followed when utilizing this tool. For each call to
+:py:meth:`PhotCTEAPI.correct_photomerty`, AWS will assemble the compute resources,
+install the software, compute the CTE corrections, send the result back,
+and then terminate all resources. Hence, users should try to process as many
+sources as they can in a single function call. Testing has shown that the
+optimal number is < 25000, but the service can handle as many as 150,000
+sources in a single request.
+
+
+Example
+-------
+Get the CTE corrected FLT photometry for a list of 1000 sources:
+
+>>> import numpy as np
+>>> from acstools import acsphotcte
+>>> ytransfers = 2048 * np.random.random(size=1000)
+>>> fluxes = 20000*np.random.random(size=len(ytransfers))
+>>> local_skys = 80*np.random.random(size=len(ytransfers))
```

Comment:
It will always be 1,000. I just did that so if I could see how well it handled 1 vs 10 vs 100 vs 1,000 vs 10,000 vs 100,000 without having to change it in 3 different places every time.

Reply:
Why not do a `n_sample = 1000` and then re-use `n_sample`?